addons/common/fnc_getPos.sqf

```diff
@@ -31,6 +31,10 @@ params [
     ["_entity", objNull, [objNull, grpNull, "", locationNull, taskNull, [], 0]] // [] and 0 to handle position
 ];
 
+if (_this isEqualType [] && {_this isEqualTypeArray [grpNull, 0]}) then {
+    _entity = _this;
+};
```

Comment:
What it currently not handles is someone passing it like this:
```sqf
[_waypoint] call CBA_fnc_getPos
```
which is strange, because positions *must* be passed like this:
```sqf
[_position] call CBA_fnc_getPos
```

Reply:
what? it handles both ways
`[_waypoint] call CBA_fnc_getPos;` and `_waypoint call CBA_fnc_getPos;`
L55 handles that